liblangutil/ParserBase.cpp

```diff
@@ -85,6 +85,45 @@ void ParserBase::expectToken(Token _value, bool _advance)
 		m_scanner->next();
 }
 
+void ParserBase::expectTokenRecoveryDelete(Token _value, bool _advance)
+{
+	Token tok = m_scanner->currentToken();
+	if (tok != _value)
+	{
+		auto tokenName = [this](Token _token)
```

Comment:
Be explicit about the return type of this lambda expression, such as: `auto tokenName = [this](Token _token) -> string { ... };`

Reply:
This is in code that was there before. I am not totally certain exactly how you would like to change. Would you make a suggested change and then I'll incorporate that into testing? 

Or perhaps this should be in a separate PR?

Reply:
I think it is fine to do that within this PR; it's small enough.

Reply:
Ok. Since you know what you are talking about and I am not confident I do, would you then suggest the small change here. I'll test it. Thanks.